runtime/src/assets/stylesheets/shared/_custom-mixins.less

```diff
@@ -0,0 +1,73 @@
+@import (reference) './../../../../node_modules/patternfly/node_modules/bootstrap/less/mixins';
+@import (reference) './../../../../node_modules/patternfly/src/less/mixins';
+//@import 'patternfly/mixin_overrides';
+
+// functions
+/* CSS3 calc() function to perform calculations */
+/* this is temporary until Less conversion is completed */
+.calc(@property; @val) {
+  @{property}: calc(~'@{val}');
+  @{property}: -moz-calc(~'@{val}');
+  @{property}: -webkit-calc(~'@{val}');
+  @{property}: -o-calc(~'@{val}');
+}
+//custom mixins
+.transform(@transform) {
+  -webkit-transform: @transform;
```

Comment:
I noticed that you have the `less-plugin-autoprefix` package installed, but are not using it in the gulp build. Can that be utilized so that all of these prefixes can be removed and the file cleaned up?